src/components/PressableOpacity.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactElement, ReactNode, useState } from 'react'
+import { AccessibilityRole, StyleProp, ViewStyle } from 'react-native'
+import styled from 'styled-components/native'
+
+interface Props {
+  children: ReactNode
+  onPress: () => void
+  style?: StyleProp<ViewStyle>
+  testID?: string
+  accessibilityRole?: AccessibilityRole
```

Comment:
I think we don't use accessibility props in lunes.

Reply:
maybe the we could use a testID. I just kept them for not failing the tests